src/types/checks.ts

```diff
@@ -0,0 +1,3 @@
+export type CheckIDsMap = {
+  [_: string]: boolean
+}
```

Comment:
small nit, but for consistency I think: 
```suggestion
  [key: string]: boolean
```
is preferred. If you regex search the codebase for `\[_:` no matches come up. If you search for `\[key:` you'll see that this is a much more common pattern